examples/llm/csrc/layer_norm/ln.h

```diff
@@ -0,0 +1,242 @@
+// Copied from https://github.com/HazyResearch/flash-attention/tree/eb33e587e95ec29a13c58f76dadca04b64122784/csrc/layer_norm
```

Comment:
[strongly consider adding all of these files as integrations (or convert the to a pip install)](https://mosaicml.slack.com/archives/C02KA7Y7NH0/p1678137737349809) instead of copy pasting thousands of lines of code which we will then need to maintain